src/components/button/bl-button.stories.mdx

```diff
@@ -209,6 +204,41 @@ We have 2 types of disabled buttons: **Contained** and **Text**. Disable version
   </Story>
 </Canvas>
 
+## Hover States
+
+export const SingleButtonHoverTemplate = (args) => html`<bl-button
+    class="__ONLY_FOR_STORYBOOK_DEMONSTRATION_HOVER__"
+    variant=${ifDefined(args.variant)}
+    kind=${ifDefined(args.kind)}
+    href=${ifDefined(args.href)}
+    target=${ifDefined(args.target)}
+    size=${ifDefined(args.size)}
+    icon="${ifDefined(args.icon)}"
+    label="${ifDefined(args.label)}"
+    ?disabled=${args.disabled}
+    style=${ifDefined(args.styles ? styleMap(args.styles) : undefined)}
+      >${unsafeHTML(args.content)}</bl-button>`
```

Comment:
To be able to do this I should implement a parameter in the `SingleButtonTemplate` which I don't want to do at first. However, `class=${ifDefined(args.class)}` may have been considered because it won't affect other stories thanks to `ifDefined`.

Reply:
I thought same. I think it's better now.